operations/finance.md

```diff
@@ -620,3 +620,85 @@ Adds a new external payment to a bill of the specified customer. An external pay
 {}
 ```
 
+## Get all preauthorizations by customers
+
+Get all customers preauthorizations.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/preauthorizations/getAllByCustomers`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "CustomerIds": ["49aaff6b-32d8-48f5-8234-ce875aefc508"]
```

Comment:
Format this just like other arrays, use data from demo in the request at least so that you can use tha API call and successfully run on demo.